addons/account_edi/models/account_edi_format.py

```diff
@@ -14,6 +14,10 @@
 import logging
 import pathlib
 import re
+try:
+    from PyPDF2.errors import PdfReadError
+except ImportError:
+    from PyPDF2.utils import PdfReadError
```

Comment:
```suggestion
from PyPDF2.utils import PdfReadError
```
Couldn't we just do this?

Reply:
Hello @epictete, changes have been done.